### PR TITLE
Portfolio site pair

### PR DESCRIPTION
**Pull request by Becky449**
#Contact section on the mobile version
- Added form content on footer of the portfolio mobile site.
- linked submit button to formspree.io account
- added styling in relation to figma template 1

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -369,3 +368,97 @@
...
+  height: 48px;
+  background: #fff;
+  border-radius: 8px;
+  margin: 22px 211px 0 0;
+}
+
+.btn:hover {
+  background-color: #6070ff !important;
+  transition: all 0.5s ease-in-out;
+  color: #fff !important;
```
Comment (prolajumokeoni) [inline styles.css]: - [ ] Kindly avoid overusing `important` in your project, it is not a best practice 👍🏿

File: index.html [hunk 2]
```diff
@@ -150,9 +150,29 @@ <h2 class="content-text">About
...
+                    <h2>
+                        Contact me
+                    </h2>
+                    <p>
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                    <form action="https://formspree.io/f/mknegeno" method="POST">
+                        <input class="name" name="Name" type="text" maxlength="30" placeholder="Becky" required>
+                        <input class="email" type="email" name="email" placeholder="becky@gmail.com" required>
+                        <textarea placeholder="Write Your Message Here" id="sheet" name="message" cols="30" rows="10" maxlength="500" required></textarea>
+                        <button class="btn" value="submit" type="submit">
```
Comment (prolajumokeoni) [inline index.html:165]: - [ ] Kindly add button interactions for active (when it's pressed),  as shown in the Figma template 👍🏿 

Screenshot_421

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Becky449 and @Osoro254Alex 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi team 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎯 
✔️ Used correct GitHub Flow
✔️ Good PR title and Description
✔️ No linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**